test/envtest/suite_test.go

```diff
@@ -59,57 +62,66 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
-	logf.SetLogger(klogr.New())
-	ctx, cancel = context.WithCancel(context.TODO())
-
 	By("bootstrapping test environment")
+
 	testEnv = &envtest.Environment{
 		CRDDirectoryPaths: []string{filepath.Join("..", "..", "deployments", "operator", "crd", "bases")},
 	}
-
-	var err error
 	cfg, err = testEnv.Start()
-	Expect(err).ToNot(HaveOccurred())
-	Expect(cfg).ToNot(BeNil())
 
-	err = devicepluginv1.AddToScheme(scheme.Scheme)
-	Expect(err).NotTo(HaveOccurred())
+	Expect(err == nil && cfg != nil)
+}, 60)
 
-	// +kubebuilder:scaffold:scheme
+var _ = AfterSuite(func() {
+	By("tearing down the test environment")
 
-	k8sManager, err = ctrl.NewManager(cfg, ctrl.Options{
+	Expect(testEnv.Stop() == nil)
+})
+
+func up(kind string, upgrade bool) {
+	logf.SetLogger(klogr.NewWithOptions(klogr.WithFormat(klogr.FormatKlog)))
+
+	ctx, cancel = context.WithCancel(context.TODO())
+
+	Expect(devicepluginv1.AddToScheme(scheme.Scheme) == nil)
+
+	// +kubebuilder:scaffold:scheme
```

Comment:
The code in this file used to be auto-generated by kubebuilder scaffolding. I guess this tag can be dropped now.